content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all external services by default
+
+An alternative to calling external services directly is to instruct the Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exists inside
+the mesh and no HTTP/TCP `ServiceEntry` is defined. For example, if your mesh does not have an HTTP service on port 443,
+and you did not define a `ServiceEntry` on port 443, you can access any external service on port 443. Note, however,
+that once you create an HTTP service on port 443 or define any `ServiceEntry` for any host on
+port 443, all accesses to port 443 is blocked: Istio will fall back to the blocking-by-default behavior for that
+port only. (Defining an HTTP service on port 443 is not recommended anyway, since using the same port for TCP/HTTPS and
+for HTTP traffic in Istio is discouraged.)
+
+1.  Make a couple of requests to external HTTPS services from `SOURCE_POD` to verify that they are blocked (it means you
+    did not configure `ServiceEntry` and `VirtualService` for the external services and that you did not enable
+    direct access to external services).
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://www.google.com | grep -o "<title>.*</title>"; kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://edition.cnn.com | grep -o "<title>.*</title>"
+    command terminated with exit code 35
+    command terminated with exit code 35
+    {{< /text >}}
```

Comment:
I think we don't need this step here in the flow, it is confusing....I would expect to get to the install option right away... because you have a step later on change back to block all traffic and user will do this step again in there.

Reply:
Fixed by c2f50714fd696076f534ba361213c689c01834bb.